packages/core/core/src/services/metrics/middleware.ts

```diff
@@ -19,7 +19,11 @@ const createMiddleware = ({ sendEvent }: { sendEvent: Sender }) => {
   const middleware: Core.MiddlewareHandler = async (ctx, next) => {
     const { url, method } = ctx.request;
 
-    if (!url.includes('.') && ['GET', 'PUT', 'POST', 'DELETE'].includes(method)) {
+    if (
+      !url.includes('.') &&
+      url.includes('/api') &&
```

Comment:
Instead of hard-coding the `'/api'` prefix, you can get it dynamically from the REST API config.

You can access it like this

```typescript
interface APIConfig {
  rest: {
    prefix: string;
    // ... other props
  }
}

const { rest } = strapi.config.get('api');

console.log(rest.prefix);
```